_includes/config-guide/custom-logger-handler-examples.md

```diff
@@ -0,0 +1,157 @@
+You can use one of the following approaches for logging into custom file:
+- setup custom log file in the `di.xml`
+- setup custom file in the custom logger handler class 
+
+## Setup custom log file in the `di.xml`
+
+1. Define custom log file in the virtual type
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualDebug" type="Magento\Framework\Logger\Handler\Base">
+    <arguments>
+        <argument name="fileName" xsi:type="string">/var/log/payment.log</argument>
+    </arguments>
+</virtualType>
+```
+
+1. Specify handler which will be used
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualLogger" type="Magento\Framework\Logger\Monolog">
```

Comment:
I would ask you to reconsider the "VirtualLogger" name since it might be a little bit confusing. Technically, it's a real logger, just pre-generated. 
It's not a big deal, but might be a point of confusion for some readers.

Reply:
So I propose some class name without "Virtual" prefix

Reply:
`Magento\Payment\Model\Method\VirtualLogger` is the example from the [Magento codebase](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Payment/etc/di.xml#L81). But, I've **implemented** your suggestions in a reason of better explanations.

Thank you!